### PR TITLE
Station Location translation

### PR DESCRIPTION
**Pull request by HB9HIL**
Goooood Morning

I started translating the Station Location Menu..

Question: 
In the code sometimes "Station Profile" and sometimes "Station Location" is used.. In my eyes it is the same. Should I establish continuity? I think "Station Profile" would be the more correct Name for it. 

What do you think?

### CODE REVIEW

**Comment by HB9HIL** [comment]
For example here:   `application/controllers/Station.php from Ln 64`

grafik

**Review by int2001 — APPROVED** [review]
Great work. Tested this.

**Comment by int2001** [comment]
> Question: In the code sometimes "Station Profile" and sometimes "Station Location" is used.. In my eyes it is the same. Should I establish continuity? I think "Station Profile" would be the more correct Name for it.
> 
> What do you think?

You're absolutely right. In my opinion i'd prefer Station Profile.
Further opinions? @AndreasK79 @phl0 @magicbug

**Comment by HB9HIL** [comment]
> Great work. Tested this.

I will also edit edit.php and create.php for this PR 👍

**Comment by AndreasK79** [comment]
Well, the table is named station_profile, but inside Cloudlog, it is called station location. I think Pete changed the naming a while back. I think he is the one that needs to decide this one.

**Comment by HB9HIL** [comment]
PS: This will be a bigger PR because just the Station Location / Profile Section is pretty much to add the multilanguage support.. 😄 Just for you to be prepared

File: application/language/polish/general_words_lang.php
Comment (HB9HIL) [inline application/language/polish/general_words_lang.php:1]: Seems like I forgot to sync this file with english.

**Comment by magicbug** [comment]
I changed it from Profile to Location as it seemed to cause confusion for a lot of people

**Comment by HB9HIL** [comment]
Okay. I didn't change Station Location to Station Profile for now. Maybe we can check this later again. But the Station Location Menu now have multilanguage Support

**Review by int2001 — APPROVED** [review]
tested this. runs without bugs. Great work, @HB9HIL 
in my eyes ready 2 merge

**Comment by magicbug** [comment]
Sorry watching AMSAT-UK Conference stream but going through PRs merged :D

**Pull request merged**